components/navigation/navChild.js

```diff
@@ -65,24 +65,53 @@ const NavChild = ({ slug, page, color, className }) => {
   }
 
   let link;
+  let divider;
   let icon;
   let target;
-  let url = page.url;
+  let url;
+  if (typeof page.url !== "undefined") {
+    url = page.url;
+  } else {
+    url = "";
+  }
 
-  const isLocalPage = page.url.startsWith("/");
+  const isLocalPage = url.startsWith("/");
+  const isCrossLinkedPage = url.startsWith("https://docs.streamlit.io");
+  const isDivider = url === "";
```

Comment:
It might be fine for now, but I feel it's a bit too hacky to fake a divider if the url is empty 😄 I feel it can lead to unexpected behavior in the future. Maybe we add a comment explaining why we're doing it this way, with a `TODO` to fix it later on?

Reply:
I can change it if you'd prefer a new parameter. I thought this was slick since it avoids cluttering things up with more parameters and reduces the burden of remembering more parameters. Forgetting to set the URL would be an error since the menu expansions are dependent on the URLs being set. Such a mistake would be visually apparent, too.

If you prefer I create a new parameter, what sounds good? Just `isDivider = true` directly in menu.md?

Reply:
I see what you're saying, and ultimately is up to you, since you'll be updating the website constantly, so I'm fine with whatever makes your work easier 👍 

If we keep this logic though, I'd recommend adding a comment either on this component, and/or in `menu.md` whenever we use `url: ""`, to explain to other people exploring the codebase why the url is empty

Reply:
I added a comment to the component.